File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.py
class Solution(object):
    def pivotArray(self, nums, pivot):
        """
        :type nums: List[int]
        :type pivot: int
        :rtype: List[int]
        """
        n = len(nums)
        lis = []

        for i in nums:
            if i < pivot:
                lis.append(i)
        
        for i in nums:
            if i == pivot:
                lis.append(i)

        for i in nums:
            if i > pivot:
                lis.append(i)

        return lis
        

File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.cpp
class Solution {
public:
    vector<int> pivotArray(vector<int>& nums, int pivot) {
        vector<int> less, more;
        int eq = 0;
        for(auto num : nums) {
            if(num < pivot) {
                less.push_back(num);
            } else if(num > pivot) {
                more.push_back(num);
            } else {
                eq++;
            }
        }
        for(int i = 0; i < eq; i++) {
            less.push_back(pivot);
        }
        for(auto num : more) {
            less.push_back(num);
        }
        return less;
    }
};